core/src/test-core/java/com/radixdlt/harness/simulation/monitors/consensus/LivenessInvariant.java

```diff
@@ -99,14 +98,14 @@ public Observable<TestInvariantError> check(RunningNetwork network) {
             emitter -> {
               nodeEvents.addListener(
                   (node, highQCUpdate) -> {
-                    emitter.onNext(highQCUpdate.getHighQC().highestQC());
+                    emitter.onNext(highQCUpdate.newHighQc().highestQC());
                   },
                   BFTHighQCUpdate.class);
               nodeEvents.addListener(
-                  (node, committed) -> {
-                    emitter.onNext(committed.vertexStoreState().getHighQC().highestQC());
+                  (node, highQcUpdate) -> {
+                    emitter.onNext(highQcUpdate.newHighQc().highestQC());
                   },
-                  BFTCommittedUpdate.class);
+                  BFTHighQCUpdate.class);
```

Comment:
I think this code is now a duplicate so should be deleted?

Reply:
yup, good catch